### DISCUSSION TITLE
[Photos] Option to name album during upload even if there are subfolders

### DISCUSSION THREAD

**Discussion by oxfighterjet** [ente-io/ente#1270] [Enhancements]
### Description

Thank you for Ente Photos. I love it. I just have a minor piece of feedback regarding the folder upload functionality. Description follows:

### Description
When uploading local folders of photos, it is possible to upload to a new album which can be named as wished. However, if the folder contains subfolders, for some reason, ente does not offer the possibility of naming the folder, instead asks whether to upload to a new folder and immediately after that starts uploading. What happens is that it reuses the name of the local folder as album name. But does not offer an option to name it before uploading.

### Expected outcome
When uploading a folder, no matter whether the folder has subfolders or not, I was expecting Ente Photos to offer me the option to upload to a new Album and if selected, to be able to name that album as I wish.

### Version
This concerns Ente Photos desktop, using the RPM build, version 1.6.63 from here: https://github.com/ente-io/photos-desktop/releases/tag/v1.6.63

### What product are you using?

Ente Photos

### What platform are you using?

Desktop - Linux (Fedora 39)

**Comment by mnvr** [comment]
Thank you for the detailed description. This'll help us, we'll see if we can restructure the upload flow to cater to this (ps I moved this to discussions, we're keeping track of non-bugs here).

> Thank you for Ente Photos. I love it.

❤️ Thank you!
